1680122527062	Sync.LogManager	DEBUG	Flushing file log
1680122527064	Sync.LogManager	DEBUG	Log cleanup threshold time: 1679258527064
1680122527065	Sync.LogManager	DEBUG	Done deleting files.
1680122950551	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1680122950551	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1680122950554	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1680122950556	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1680123455201	Sync.SyncScheduler	DEBUG	Woke from sleep.
1680123455208	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1680123455211	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1680123455264	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1680123457214	Sync.Service	DEBUG	User-Agent: Firefox/111.0.1 (Linux x86_64) FxSync/1.113.0.20230323121228.desktop
1680123457214	Sync.Service	INFO	Starting sync at 2023-03-29 16:57:37 in browser session gA1_ybwwqL3P
1680123457216	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1680123457232	Sync.Status	INFO	Resetting Status.
1680123457232	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1680123457236	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1680123457236	FirefoxAccounts	DEBUG	getOAuthToken enter
1680123457238	FirefoxAccounts	TRACE	getCachedToken returning cached token
1680123457238	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1680123457241	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1680123457250	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1680123457251	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 491\"  data: no]"}
1680123457251	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/199676341/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1680123457252	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1680123457252	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1680123457252	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1680123457252	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1680123457256	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
